spec/models/miq_expression_spec.rb

```diff
@@ -92,7 +92,7 @@
       exp1 = {"STARTS WITH" => {"field" => "Vm-name", "value" => "foo"}}
       exp2 = {"ENDS WITH" => {"field" => "Vm-name", "value" => "bar"}}
       sql, * = MiqExpression.new("OR" => [exp1, exp2]).to_sql
-      expect(sql).to eq("\"vms\".\"name\" LIKE 'foo%' OR \"vms\".\"name\" LIKE '%bar'")
+      expect(sql).to eq("(\"vms\".\"name\" LIKE 'foo%' OR \"vms\".\"name\" LIKE '%bar')")
```

Comment:
To clarify, do we need to ensure all subclasses of Expression implement non-abstract methods?  Should we make sure the timezone specific components do the right thing with respect to timezones?  Do we need to have tests that demonstrate the parsing of `@pexp` into the right subclasses?